src/Commands/AppConfig/KeyValue/KeyValueDeleteCommand.cs

```diff
@@ -29,12 +26,16 @@ Delete a key-value pair from an App Configuration store. This command removes th
     [McpServerTool(Destructive = true, ReadOnly = false, Title = _commandTitle)]
     public override async Task<CommandResponse> ExecuteAsync(CommandContext context, ParseResult parseResult)
     {
-        var args = BindArguments(parseResult);
+        var args = BindOptions(parseResult);
 
         try
         {
-            if (!await ProcessArguments(context, args))
+            var validationResult = Validate(parseResult.CommandResult);
```

Comment:
This validation logic is similar in most of the commands, is it possible to push move it up a level to the command handler?